src/util.c

```diff
@@ -226,10 +226,12 @@ jq_util_input_state *jq_util_input_init(jq_util_msg_cb err_cb, void *err_cb_data
     err_cb_data = stderr;
   }
   jq_util_input_state *new_state = jv_mem_calloc(1, sizeof(*new_state));
-  new_state->err_cb = err_cb;
-  new_state->err_cb_data = err_cb_data;
-  new_state->slurped = jv_invalid();
-  new_state->current_filename = jv_invalid();
+  if(new_state) {
```

Comment:
Space between `if` and `(`?

Reply:
Hmm, yeah, sorry, I had my editor configured for `curl`-style